site/downloads.markdown

```diff
@@ -77,7 +77,7 @@ Alternatively, many operating systems provide GHC, cabal and Stack through their
 <div id="collapse-nixos" class="collapse">
 <!-- This installation method is owned by @maralorn -->
 
-*for Nix the package manager on the distribution NixOS and other OS*
+*For Nix the package manager on the distribution NixOS and other OS:*
```

Comment:
```suggestion
*[Nix] is the package manager for NixOS and can also be used on other Linux distros and macOS.*

[Nix]: https://nixos.org
```

@maralorn I like your suggestion for making the context a bit more explicit. If we're doing that, seems like it's worth linking to the Nix website and mentioning macOS explicitly.